CIP-????/README.md

```diff
@@ -0,0 +1,250 @@
+
+Governance is difficult. Discussions on CIP-1694 can attest to that quite clearly. There are constant debates within the Cardano community about changing protocol parameters, and the decision ultimately falls -- at this stage still -- onto the three genesis entities: Input Output, The Cardano Foundation and Emurgo. Yet, at this stage, few governance tools are at their disposal to make educated decisions. Besides Twitter polls, newsletter surveys, and SPO town halls on Discord, we have identified a gap and an opportunity to engage with the Cardano community through means currently at our disposal.
+
+Conducting an on-chain poll between SPOs can also be seen as an experiment and an evaluation of the network's participation and engagement in the governance questions. Even though we only propose to poll one particular group of the Cardano community (the SPOs), such events can help to provide actual data to fuel the conversations around CIP-1694.
+
+In summary, the goals are:
+
+1. [x] to make some first experimental baby steps in the realm of governance;
+1. [x] to capture participation data from SPOs;
+1. [x] to raise awareness amongst SPOs regarding their future role in governance;
+1. [x] to keep the Voltaire dynamics up in the ecosystem while other efforts are being pursued;
+1. [x] to improve relations between the Cardano Foundation & SPOs for better mutual understanding and fruitful conversations.
+
+
+## Specification
+
+### Overview
+
+Polls will be multiple-choice questions by The Cardano Foundation with pre-defined answers to choose from.
+
+Here's an example of a question and answers:
+
+- _Pineapples on pizza?_
+  - [ ] yes
+  - [ ] no
+
+The serialised question and answers will be posted on-chain and signed by one of the delegate genesis keys owned by The Cardano Foundation. Answers will be provided on-chain by participating SPOs via transaction metadata referring to:
+
+- The question and answers
+- The index of the chosen answer from the available choices
+- A signature from the SPO's current VRF credentials or Ed25519 declared cold credentials.
+
+> **Note**
+> In this document, every time we refer to a _serialized object_, we refer to its **canonical** CBOR representation. In particular, keys in a map are always ordered alphabetically.
+
+### Question structure
+
+A question is posted in a transaction's metadata using the metadata label `94` and the following metadata structure:
+
+```cbor
+question =
+  { 0: prompt
+  , 1: [ * choice ]
+  , ? "_": nonce
+  }
+
+prompt =
+  text .size (0..64) / [ * text .size (0..64) ]
+
+choice =
+  text .size (0..64) / [ * text .size (0..64) ]
+
+nonce =
+  uint
+```
```

Comment:
```suggestion
```cbor
question =
  { 0: prompt
  , 1: [ * choice ]
  }

prompt =
  text .size (0..64) / [ * text .size (0..64) ]

choice =
  text .size (0..64) / [ * text .size (0..64) ]

nonce =
  uint
  
poll = 
{ 0 : [ + question ]
  , ? "_" : nonce
}
```
```

With a minor change to a master `poll` object that can contain one-or-more questions the flexibility of this system can be greatly enhanced to support multiple-question polling while utilizing all of the same mechanics otherwise described. Here the nonce is added at the top-level to de-duplicate.

Reply:
I am not fond of this added flexibility; because it is also extra complexity for tooling down the line. But more importantly, I don't think we will ever want to ask multiple questions at once. Perhaps the example in the CIP is too silly to make this obvious but, we ain't going to ask about pineapples on pizzas. We need people to take time and do proper research on questions asked; so asking more than one question at a time is a recipe for disaster IMO.